types.d.ts

```diff
@@ -2,9 +2,8 @@ import { EventEmitter } from "events";
 import https from 'https';
 import * as core from 'express-serve-static-core'
 
-//TODO: need to better understand this type
-type InputType = {}
-
+//TODO: https://iot.mozilla.org/wot/#example-7-action-object seems to imply input could be typed?
```

Comment:
@hobinjk should InputType be strongly typed based on the information in the URL mentioned? Or should we leave as any?

Reply:
It should be left as `any` since otherwise we'd need to describe the entirety of [JSON schema](https://json-schema.org/) in an annotation which seems overly difficult